meson.build

```diff
@@ -733,12 +733,10 @@ if features['sdl2-gamepad']
     sources += files('input/sdl_gamepad.c')
 endif
 
-stdatomic_dep = cc.find_library('atomic', required: false)
+stdatomic_dep = cc.find_library('atomic', required: true)
```

Comment:
This will break BSDs that don't actually use the stdatomic library and have it as a part of compiler-rt so it should still be false.